src/main/rx/srxl2.c

```diff
@@ -0,0 +1,545 @@
+#include "platform.h"
+
+#include "common/crc.h"
+#include "common/maths.h"
+#include "common/streambuf.h"
+
+#include "drivers/exti.h"
+#include "drivers/nvic.h"
+#include "drivers/time.h"
+
+#include "io/serial.h"
+
+#include "rx/srxl2.h"
+#include "rx/srxl2_types.h"
+#include "io/spektrum_vtx_control.h"
+
+#include "drivers/serial.h"
+#include "drivers/serial_uart.h"
+
+#include <string.h>
+
+#ifndef SRXL2_DEBUG
+#define SRXL2_DEBUG 0
+#endif
+
+#if SRXL2_DEBUG
+//void cliPrintf(const char *format, ...);
+//#define DEBUG(format, ...) cliPrintf(format, __VA_ARGS__)
+#define DEBUG(...) //Temporary until a better debug printf can be included
+#else
+#define DEBUG(...)
+#endif
+
+#ifdef USE_SERIALRX_SRXL2
```

Comment:
This should go right underneath `platform.h` (which includes the target specific defines).

Reply:
Done